src/coinbase/address.ts

```diff
@@ -295,6 +297,31 @@ export class Address {
     );
     return new FaucetTransaction(response.data);
   }
+  /**
+   * Returns the reputation of the Address.
+   *
+   * @returns The reputation of the Address.
+   * @throws {Error} if the API request to get the Address reputation fails.
+   * @throws {Error} if the Address reputation is not available.
+   */
+  public async reputation(): Promise<AddressReputation> {
+    const response = await Coinbase.apiClients.addressReputation!.getAddressReputation(
+      this.getNetworkId(),
+      this.getId(),
+    );
+
+    this._reputation = new AddressReputation(response.data);
```

Comment:
If we've previously fetched reputation, should we re-fetch it every time this is called?

Reply:
Added a check if it been already fetched then it will not be fetched again